packages/documentation/docs/installation/vue.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 2
+sidebar_title: Vue
+title: Vue
+---
+
```

Comment:
We should also add here a disclaimer that what `experimental` mean for us

Reply:
what would that explanation be?

Reply:
@nuke-ellington any suggestions how we should describe the experimental state of an wrapper component?

What we will not do for experimental wrappers are:

- Blueprints
- Support
- Bugfixes after react/angular
- ...